crates/evm/src/evm/handler.rs

```diff
@@ -23,18 +24,23 @@ use tracing::instrument;
 use crate::system_events::SYSTEM_SIGNER;
 use crate::{BASE_FEE_VAULT, L1_FEE_VAULT};
 
-const DB_ACCOUNT_SIZE: usize = 74;
+/// Eoa size is reduced because code_hash for eoas are None on state diff, converted to empty Keccak  internally for evm operations
+const DB_ACCOUNT_SIZE_EOA: usize = 42;
```

Comment:
74 - 32 + 1 (for none) = 43???

Reply:
1 byte len descriptor for code hash

Reply:
But option<T> 's size is 1 on none

Reply:
It should be 73 but there is 1 byte length descriptor for code hash 0x20 + code hash
0x20 is the 1 extra byte